pkg/handlers/redirect.go

```diff
@@ -22,17 +22,23 @@ import (
 	"strings"
 )
 
-// HandleHeaderRedirect redirct the requests to configured auth URL by setting Location header and sending 302.
+// HandleHeaderRedirect redirect the requests to configured auth URL by setting Location header and sending 302.
 func HandleHeaderRedirect(w http.ResponseWriter, r *http.Request, opts map[string]interface{}) {
-	authURLPath, sep, redirectParameter, redirectURL, redirect := redirectParameters(w, r, opts)
+	authURLPath, sep, redirectParameter, redirectURL, loginHint, redirect := redirectParameters(w, r, opts)
```

Comment:
@PedroDBFlores , the the hint is already in `opts`. Thus, no need to create the `loginHint` variable.